packages/react-components/react-migration-v0-v9/src/components/Button/ButtonMigration.mixins.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Curious what triggers that rule 🤔

Reply:
It was the `style` object, but since we will be using matchSnapshot, that is not needed anymore